vhdbuilder/packer/install-dependencies.sh

```diff
@@ -68,6 +61,13 @@ cat << EOF >> ${VHD_LOGS_FILEPATH}
   - zip
 EOF
 
+if [[ ${UBUNTU_RELEASE} == "18.04" && ${ENABLE_FIPS,,} == "true" ]]; then
+  installFIPS
+elif [[ ${ENABLE_FIPS,,} == "true" ]]; then
+  echo "AKS enables FIPS on Ubuntu 18.04 only, exiting..."
+  exit 1
+fi
+
```

Comment:
Does this mean FIPS has impact to DNS resolution for some URL?

Reply:
As we talked offline and I also post in the channel, FIPS should not have impact to DNS resolution for some URL. And also during VHD build time, FIPS kernel/packages are just installed but has not boot into, so FIPS is not in effect at that time. According to my investigation, from March 18 or 19, apt_get_dist_upgrade() starts to install some systemd related packages, and the NS in /run/systemd/resolv/resolv.conf is wiped out during this installation. Anyway, changing the installation order of FIPS and apt_get_dist_upgrade() fix the issue starting to happen from March 18 or 19.